app/src/main/java/is/xyz/mpv/MPVView.kt

```diff
@@ -109,6 +109,16 @@ internal class MPVView(context: Context, attrs: AttributeSet) : SurfaceView(cont
             MPVLib.setOptionString("vd-lavc-skiploopfilter", "nonkey")
         }
 
+        var ytdl_format = sharedPreferences.getString("ytdl_format", "")
```

Comment:
Either `video_ytdl_format` or `ytdl_format`. Using one key here and the other in the preferences won't work. Pick one. I suggest renaming the preference key to `ytdl_format`, because it isn't just video-related.

Reply:
Oh great, wonder how I messed this one up.

Reply:
Oh great, wonder how I messed this one up.